### PR TITLE
Minor error preventing phone animationto display

### PR DESCRIPTION
**Pull request by josephprasanth**

### CODE REVIEW

File: app/templates/index.html [hunk 1]
```diff
@@ -30,10 +30,14 @@ <h3 class="mbr-header-text">Follow all your lifestyle interests in one <span cla
             </div>
             <div class="mbr-box-magnet mbr-box-magnet-top-left mbr-section-right mockup-frames col-sm-8">
               <div class="mbr-figure-container phone-container">
+                <p>Test1111 </p>
```
Comment (klokie) [inline app/templates/index.html:33]: what is this?

File: app/templates/index.html [hunk 2]
```diff
@@ -30,10 +30,14 @@ <h3 class="mbr-header-text">Follow all your lifestyle interests in one <span cla
...
+
                 <figure class="mbr-figure mbr-figure-adapted mbr-figure-caption-inside-bottom mbr-figure-full-width"><img  class="mbr-figure-img frame-phone" src="/images/frame-phone.png" width="162" height="301" alt="phone"/>
                   <div class="flex-slider flex-slider-phone" data-slider="phone">
+                    <p>Test222 </p>
```
Comment (klokie) [inline app/templates/index.html:37]: what is this?

File: app/templates/index.html (hunk 1 above)
Comment (josephprasanth) [reply app/templates/index.html:33]: I just added it to show you that not even the p comment  is being displayed in the section. Do you think that 2 different animations on both the ipad and iphone had some issues running at the same time

**Comment by klokie** [comment]
@josephprasanth, I can't evaluate this patch until you've committed all the new images. Could you please add the three images you worked on to the `images/mockups` directory.
You've called them `1-new.jpg`, `5-new.jpg`, and  `6-new.jpg`, although I'd prefer if you would just replace the existing files `1.jpg`, `5.jpg`, and  `6.jpg`. That's what Git is good for ;)
Can you please do that and resubmit the pull request?

**Comment by josephprasanth** [comment]
Hi Daniel,
sorry about this I had already uploaded it before into that section and I thought git pull would automatically upload the files to you , let me redo a git commit , with them ,
cheers ,
Joseph  

```
On Thursday, September 1, 2016 5:50 PM, Daniel « Klokie » Grossfeld <notifications@github.com> wrote:
```

 @josephprasanth, I can't evaluate this patch until you've committed all the new images. Could you please add the three images you worked on to the images/mockups directory.
You've called them 1-new.jpg, 5-new.jpg, and 6-new.jpg, although I'd prefer if you would just replace the existing files 1.jpg, 5.jpg, and 6.jpg. That's what Git is good for ;)
Can you please do that and resubmit the pull request?—

**Comment by josephprasanth** [comment]
My Init.js file , is giving an error ... I have to figure out what it is ( it happened with a merge )

$ git commit -a -m "Latest image changes"
warning: LF will be replaced by CRLF in app/scripts/_init.js.
The file will have its original line endings in your working directory.
warning: LF will be replaced by CRLF in app/templates/index.html.
The file will have its original line endings in your working directory.
[11:02:19] Using gulpfile ~\Documents\GitHub\y1\enliven-coming-soon-v3\gulpfile.js
[11:02:19] Starting 'lint'...
[11:02:19] Starting 'styles'...
[11:02:20] Starting 'scripts'...
[11:02:20]
C:\Users\Dindator\Documents\GitHub\y1\enliven-coming-soon-v3\app\scripts_init.js
  103:4  error  Parsing error: Unexpected token )

✖ 1 problem (1 error, 0 warnings)

events.js:141
      throw er; // Unhandled 'error' event
      ^
 ESLintError: Failed with 1 error

**Comment by klokie** [comment]
have you run `npm install && bower install` recently? You might need to

**Comment by josephprasanth** [comment]
Unfortunately the error still exists , ( Did both npm and bower install ) , The only thing that I changed there was the animation speed ... but it somehow merged somethings together .. Can I type in the full code of the init.js file here .... so maybe you ca have a look ( not so good at javascript yet :-) )

**Comment by josephprasanth** [comment]
[init.js.doc.docx](https://github.com/Enliven-se/enliven-coming-soon-v3/files/451666/init.js.doc.docx)

**Comment by josephprasanth** [comment]
ok dont what I did right , It has worked

**Pull request closed without merging**